extra_data/reader.py

```diff
@@ -1013,6 +1015,12 @@ def select(self, seln_or_source_glob, key_glob='*', require_all=False,
                     else:  # require_any
                         train_ids = np.union1d(train_ids, source_tids)
 
+                if len(train_ids) < (n_trains_prev / 2):
+                    n_dropped = n_trains_prev - len(train_ids)
+                    warn(f"{n_dropped}/{n_trains_prev} trains lost when filtering "
```

Comment:
A percentage could be useful here, especially in combination with it being an optional argument.

Reply:
Added the percentage. :+1: